src/components/Result/Answer/Answer.js

```diff
@@ -4,10 +4,18 @@ import classes from './Answer.scss';
 
 const Answer = ({ answer }) => (
     <div className={classes.AnswerSection}>
-        <h2 className={classes.AnswerHeader}>Twoje słowo:</h2>
-        <div className={classes.Answer}>
-            {answer}
-        </div>
+        <h2 className={classes.AnswerHeader}>
+            {answer ? 'Twoje słowo:' : 'Nie ułożyłeś słowa'}
```

Comment:
as a woman how do you feel when you read a message that has "-eś" ending?

Reply:
depending on situation. but thanks for asking :) I can fix that.